src/FastBroadcast.jl

```diff
@@ -13,6 +13,7 @@ getAxes(::Type{T}) where {T<:Tuple} = collect(T.parameters)
 
 use_fast_broadcast(_) = false
 use_fast_broadcast(::Type{<:Base.Broadcast.DefaultArrayStyle}) = true
+use_fast_broadcast(::Type{Base.Broadcast.DefaultArrayStyle{0}}) = false
```

Comment:
It might be safer to write `Base.Broadcast.DefaultArrayStyle{0}` in case Base adds more type parameters.

Reply:
Done.